<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebuggerConsole</class>
 <widget class="QWidget" name="DebuggerConsole">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string> Debugger Console</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,10">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,10">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="toolbar" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="btn_run">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btn_continue">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btn_step">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btn_step_in">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btn_step_out">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btn_stop">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btn_clear">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cmd_line"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="console">
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::NoWrap</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionContinue">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/app_data/icons/go-next.svg</normaloff>:/app_data/icons/go-next.svg</iconset>
   </property>
   <property name="text">
    <string>Continue</string>
   </property>
   <property name="toolTip">
    <string>Continue F7</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="actionStep">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/app_data/icons/go-last.svg</normaloff>:/app_data/icons/go-last.svg</iconset>
   </property>
   <property name="text">
    <string>Step</string>
   </property>
   <property name="toolTip">
    <string>Step F6</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionStep_into">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/app_data/icons/go-jump.svg</normaloff>:/app_data/icons/go-jump.svg</iconset>
   </property>
   <property name="text">
    <string>Step into</string>
   </property>
   <property name="toolTip">
    <string>Step into F5</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionStep_Out">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/app_data/icons/go-stepout.svg</normaloff>:/app_data/icons/go-stepout.svg</iconset>
   </property>
   <property name="text">
    <string>Step Out</string>
   </property>
   <property name="toolTip">
    <string>Step Out F8</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/app_data/icons/gtk-media-stop.svg</normaloff>:/app_data/icons/gtk-media-stop.svg</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop F11</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/app_data/icons/media-seek-forward.svg</normaloff>:/app_data/icons/media-seek-forward.svg</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="toolTip">
    <string>Run Without Debugger</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/app_data/icons/gtk-clear.svg</normaloff>:/app_data/icons/gtk-clear.svg</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="toolTip">
    <string>Clear</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
